hexrd/ui/main_window.py

```diff
@@ -486,6 +488,34 @@ def run_apply_polar_mask(self, line_data):
         HexrdConfig().polar_masks_line_data.append(line_data.copy())
         self.update_all()
 
+    def on_action_edit_apply_laue_mask_to_polar_triggered(self):
+        if not HexrdConfig().show_overlays:
+            msg = 'Overlays are not displayed'
+            QMessageBox.critical(self.ui, 'HEXRD', msg)
+            return
+
+        all_overlays = HexrdConfig().overlays
+        laue_overlays = [x for x in all_overlays if x['type'] == 'laue']
```

Comment:
`laue` ideally should be a constant, but that can come in a follow up PR if necessary.